src/pages/why.tsx

```diff
@@ -0,0 +1,412 @@
+                                        get what I mean. It’s more honest and not marketing for the sake of it - it
+                                        actively helps developers build successful products. That’s our mission.
+                                    </li>
+                                    <li>
+                                        PostHog has zero intention of selling our business. We want to see just how
+                                        crazy huge this gigantic software stack can get - and we think that it will
+                                        reach at least $100bn in value. We’ll be around and fighting for a long, long
+                                        time. It’s our life’s work.
+                                    </li>
+                                    <li>
+                                        You can also see how our entire company operates - you can see our{' '}
+                                        <Link href="/handbook/growth/sales/overview">manual for our salespeople</Link>,{' '}
+                                        <Link href="/handbook/growth/marketing">marketing team</Link>, and so on. You
+                                        can even <Link href="/handbook/people/compensation">see what they’re paid</Link>{' '}
+                                        - it’s all in our public handbook. Even the contentious stuff – there’s a reason
+                                        most companies hide this stuff!
+                                    </li>
+                                    <li>So many hedgehogs.</li>
+                                </ul>
+
+                                <p>
+                                    The reason we uniquely love doing all these sorts of things is that{' '}
+                                    <strong>
+                                        we grow because of our reputation on the internet whereas every single
+                                        competitor grows by salespeople.
+                                    </strong>{' '}
+                                    We are very, very proudly inbound only. This aligns us with customers. Long term,
+                                    that is what wins.
+                                </p>
+
+                                <h2 id="support-from-actually-technical-people">
+                                    Support from actually technical people
+                                </h2>
+                                <p>
+                                    You can <Link href="/teams">literally see the engineers</Link> that built each of
+                                    our products on our website.
+                                </p>
+
+                                <div className="@lg:float-right ml-2 @lg:ml-8 mb-6 mt-2 @lg:mb-2 px-2 py-4 max-w-sm rounded border border-light dark:border-dark bg-accent dark:bg-accent-dark">
```

Comment:
I think it would be better if this component was just full-width by default. Would avoid a lot of awkward text reflows.

Reply:
Can you screenshot specifically what you saw? I put in a non-trivial amount of effort so it looks good at just about every size screen.